packages/components/src/components/interface/Header/Mobile/MobileHeader.tsx

```diff
@@ -91,37 +91,41 @@ class MobileHeader extends React.Component<IMobileHeaderProps & IDomProps> {
     const { id, mobileLeft, mobileRight, title, mobileBody } = this.props
     return (
       <HeaderContainer id={id} className={this.props.className}>
-        {mobileLeft && (
-          <EndComponentContainer>
-            <CircleButton
-              id="mobile_header_left"
-              onClick={mobileLeft.handler}
-              color="#4972BB"
-            >
-              {!mobileBody && <Hamburger />}
-              {mobileLeft.icon()}
-            </CircleButton>
-            <>{!mobileBody && mobileLeft.icon()}</>
-          </EndComponentContainer>
-        )}
+        {mobileLeft &&
+          mobileLeft.length > 0 &&
```

Comment:
Minor thing, but you don't need to check the length if you're using `.map`. If it's 0 length, then the map just returns an empty array